AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/tier/tier-edit/tier-edit.component.ts

```diff
@@ -89,6 +88,7 @@ export class TierEditComponent {
             next: (data: HttpResponseEnvelope<Tier>) => {
                 if (data && data.result) {
                     this.tier = data.result;
+                    this.tier.name == "Basic" ? (this.tier.deleteable = false) : (this.tier.deleteable = true);
```

Comment:
What is this? You couldn't have made this more complicated, could you?

```suggestion
                    this.tier.deleteable = this.tier.name == "Basic"
```